src/controllers/pegin-tx.controller.ts

```diff
@@ -67,7 +67,7 @@ export class PeginTxController {
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
           const inputsAmount = inputs.reduce((acc, curr) => ({amount: acc.amount + curr.amount}));
-          if (inputsAmount.amount - (createPeginTxData.amountToTransferInSatoshi + fee) <= 0) reject(new Error('The stored input list is has not enough amount'));
+          if (inputsAmount.amount - (createPeginTxData.amountToTransferInSatoshi + fee) < 0) reject(new Error('The stored input list is has not enough amount'));
```

Comment:
as @jeremy-then mentioned. We should be returning the value.
On top of that we should always enclose conditions/loops in curly braces
```suggestion
          if (inputsAmount.amount - (createPeginTxData.amountToTransferInSatoshi + fee) < 0) {
            return reject(new Error('The stored input list is has not enough amount'));
          }
```